src/atlclients/authInfo.ts

```diff
@@ -240,19 +252,25 @@ export function isEmptySiteInfo(a: any): boolean {
     );
 }
 
-export function isOAuthInfo(a: any): a is OAuthInfo {
-    return a && (<OAuthInfo>a).access !== undefined && (<OAuthInfo>a).refresh !== undefined;
+export function isOAuthInfo(a: AuthInfo | undefined): a is OAuthInfo {
+    // This check should be retired over time when auth info is updated to use the new type
+    const oldCheck = a && (<OAuthInfo>a).access !== undefined && (<OAuthInfo>a).refresh !== undefined;
```

Comment:
Make it clear in comments that this has not been changed for backward compatibility reasons